script/generate-examples

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# A script to generate controller examples to STDOUT.
+
+set -e
+
+ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+source "${ROOT}/script/lib/ui.sh"
+
+usage() {
+  cat <<USAGE >&2
+usage: $0 <controller|router> [options]
+
+OPTIONS:
+  -h            Show this message
+  -i IP         The external IP address [default: the IP assigned to eth0]
+  -p PORT       The port to start the resource server on [default: 12345]
+USAGE
+}
+
+main() {
+  local ip port component
+
+  case $1 in
+    controller) component=$1 ;;
+    router)  component=$1 ;;
+    *)
+      usage
+      echo "Invalid component: $1"
+      exit 2
+      ;;
+  esac
+
+  while getopts 'hi:p:' opt; do
+    case "${opt}" in
+      h)
+        usage
+        exit 1
+        ;;
+      i) ip="${OPTARG}" ;;
+      p) port="${OPTARG}" ;;
+      c) component="${OPTARG}" ;;
+      ?)
+        usage
+        exit 2
+        ;;
+    esac
+  done
+
+  ip="${ip:-$(/sbin/ifconfig eth0 \
+    | grep -oP 'inet addr:\S+' \
+    | cut -d: -f2)}"
```

Comment:
This should not be necessary: https://github.com/flynn/flynn/commit/eb3ad9be50e3f42c69a0e6e7936e47a231d50a02